### DISCUSSION TITLE
Getting Org modern to work in Custom Agenda Commands

### DISCUSSION THREAD

**Discussion by JMinyard1335** [minad/org-modern#199] [Q&A] [answered]
So I recently came across org modern when attempting to clean up the look of my org agenda views. Some background to do this I am using:

- org-modern (extra faces I can change for todo task without breaking regular org files)
- org-super-agenda (grouping and custom faces)
- ollivetti (centering)

Org-modern is a great package but it seems I have run into some issues. Since I just want org-modern to be active in agenda views and not inside of my regular org documents I added the following to the top of my config:

`(add-hook 'org-agenda-finalize-hook #'org-modern-agenda)` 

Maybe this hook does not do what I think it does but I think it should activate the org-modern-agenda anytime an org agenda buffer is launched. This hook works fine as long as I use the default org-agenda view.

![Screenshot from 2024-05-13 08-52-35](https://github.com/minad/org-modern/assets/141966459/8e0a925c-c47b-4a3d-b0aa-80c455ba34e5)

This is the default agenda view but when I then click on my custom agenda view I am greeted with this:

![Screenshot from 2024-05-13 08-53-16](https://github.com/minad/org-modern/assets/141966459/4163b90a-07e7-4674-950e-6848d6750535)

The TODO states are not the org-modern-todo face in the custom view; just the org-todo face. Is there anyone who can explain this am I misunderstanding what the hook does? Is there a way to fix this without turning on global org modern mode?

**Comment by minad** [comment]
Since this issue affects the custom agenda, it is likely a duplicate of https://github.com/minad/org-modern/discussions/211.

**Reply by JMinyard1335** [reply]
Okay Thank you

**Reply by minad** [reply, marked as answer]
Fixed in https://github.com/minad/org-modern/commit/6550abc8c7b059cfa8b1653c280dde030787a6e7.
